docs/content/users/extend/traefik-router.md

```diff
@@ -1,10 +1,10 @@
 # Router Customization and Debugging (Traefik)
 
-Traefik is the default router in DDEV v1.22+.
+Traefik is the default router in DDEV v1.22+. The older `nginx-proxy` router is deprecated and will be removed in DDEV v1.24.
```

Comment:
I think we could have a separate page for such information.
Same as https://goreleaser.com/deprecations/

Reply:
The only other things I remember us deprecating are
* `disable-http2`, which goes away with traefik
* `nginx-proxy`

I tried for years to deprecate traditional Windows, but always got significant pushback. And now it works really quite well, amazingly.

Let's do as you say when we get this ready for v1.24.